### PR TITLE
Blog: gRPConf 2023 wrap-up

### PR DESCRIPTION
**Pull request by YifeiZhuang**

### CODE REVIEW

**Comment by YifeiZhuang** [comment]
@kevinnilson  please take a look. I have limited access in the repo, so I can not click the button to you as a reviewer.

**Comment by kevinnilson** [comment]
Need to add link to observability code lab.  can you also add links such as

Our youtube channel is:
https://www.youtube.com/@grpcio

grpc meetup group
https://www.meetup.com/grpcio/

Website: 
 gRPC.io

YouTube: 
[www.youtube.com/@grpcio](http://www.youtube.com/@grpcio)

Blog: 
[grpc.io/blog](http://grpc.io/blog)

GitHub: [github.com/grpc/grpc](http://github.com/grpc/grpc)

Forum: [groups.google.com/g/grpc-io](http://groups.google.com/g/grpc-io)

https://twitter.com/grpcio

**Comment by kevinnilson** [comment]
need to add codelab.

Also need to add links such as:

Our youtube channel is:
https://www.youtube.com/@grpcio

grpc meetup group
https://www.meetup.com/grpcio/

Website: 
 gRPC.io

YouTube: 
[www.youtube.com/@grpcio](http://www.youtube.com/@grpcio)

Blog: 
[grpc.io/blog](http://grpc.io/blog)

GitHub: [github.com/grpc/grpc](http://github.com/grpc/grpc)

Forum: [groups.google.com/g/grpc-io](http://groups.google.com/g/grpc-io)

https://twitter.com/grpcio

**Comment by YifeiZhuang** [comment]
> Need to add link to observability code lab. can you also add links such as

@kevinnilson 
Observability code lab is still not available. It might be available in a week or so but not sure yet. However, this blog is time sensitive so I would prefer publishing without the code lab link. We will have the code lab link in a separate blog post along with the gRPC Day. So, could you approve this one?

File: content/en/blog/grpconf-2023-videos.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ the presentations during the day:
 Stay tuned to gRPC on many platforms for new releases, announcements and events:
 * [gRPC YouTube Channel](https://www.youtube.com/@grpcio)
 * [gRPC meetup group](https://www.meetup.com/grpcio/)
```
Comment (kevinnilson) [inline content/en/blog/grpconf-2023-videos.md:38]: capitol M

**Review by kevinnilson — COMMENTED** [review]
added some minor comments

**Review by kevinnilson — APPROVED** [review]

**Pull request merged**